ansible/playbooks/roles/repository/tasks/copy-download-requirements.yml

```diff
@@ -15,10 +15,11 @@
           - "{{ _req_files['grafana_dashboards'] }}"
           - "{{ _req_files['images'] }}"
           - "{{ _req_files['package_files'] }}"
-          - "{{ _req_files['repo_files'] }}"
           - "download-requirements.py"
           - "src/"
 
+        _repo_files: "{{ _req_files['repo_files'] }}"  # optional repo files
```

Comment:
`_repo_files` name is misleading for me since this is not e.g. `repository/files/download-requirements/repositories/x86_64/redhat-7.yml` but `repository\files\download-requirements\requirements\x86_64\redhat-7\files.yml` so I would name it e.g. `_distro_files`.